route/access_rules.go

```diff
@@ -29,11 +29,6 @@ func (t *Target) AccessDeniedHTTP(r *http.Request) bool {
 		log.Printf("[WARN] failed to parse remote address %s", host)
 	}
 
-	// check remote source and return if denied
```

Comment:
This block belongs with the block above.  The `ip` that is being checked here is the result of the parse of the remote address in the previous block.